engines/bastion_katello/app/assets/javascripts/bastion_katello/activation-keys/details/views/activation-key-repository-sets.html

```diff
@@ -16,6 +16,18 @@
            ng-model="repositorySetFilter"/>
   </div>
 
+  <div data-block="filters">
+    <label class="checkbox-inline" title="{{ 'Show all Repository Sets available to this Activation Key' | translate }}">
+      <input type="checkbox" ng-model="contentAccessModes.contentAccessModeAll" ng-change="toggleFilters()"/>
+      <span translate>Show All</span>
```

Comment:
When testing this it wasn't clear to me what "Show All" meant.  Does this mean show all that aren't in the CVE?